.isort.cfg

```diff
@@ -0,0 +1,12 @@
+[settings]
+; see https://github.com/psf/black
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+combine_as_imports=True
+use_parentheses=True
+line_length=88
+known_odoo=odoo
+known_odoo_addons=odoo.addons
+sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
+known_third_party=copier,plumbum,pytest,yaml
```

Comment:
We don't want to update repos `.isort.cfg` with this line.

Reply:
Ooops...

Reply:
Oh my local MQT clone was out of sync, you'll see some failures due to that. Wait for next push please, sorry.